src/test/resources/net/sf/jabref/importer/fileformat/BiblioscapeImporterTestCorrupt.txt

```diff
@@ -1,5 +1,5 @@
 
-teststring
```

Comment:
Does it really have to be `te`? Couldn't you use `invalidStringMakingTheImporterFail`?